### DISCUSSION TITLE
Question about signInWithApple

### DISCUSSION THREAD

**Discussion by rrothweiler1** [Flowductive/easy-firebase#20] [Q&A] [answered]
I am using the signinwithapple feature and it is working good.  What is the best way to wait for success from this function?  The first time I ran with this the global auth still had the old user info and I don't see a way to wait for the signinwithapple function to return success and or error?

**Comment by benlmyers** [comment, marked as answer]
Hi @rrothweiler1, thanks for using EasyFirebase. As of [Release 1.4.2](https://github.com/Flowductive/easy-firebase/releases/tag/1.4.2), `EasyAuth.signInWithApple(...)` now has a completion handler. Ensure your package is up-to-date, implement the handler, and if you run into any issues, let me know! 😊
